src/library.js

```diff
@@ -910,13 +910,15 @@ LibraryManager.library = {
     {{{ makeStructuralReturn(['ret', '0']) }}};
   },
 
-  llvm_ctpop_i32: function(x) {
```

Comment:
And also, where I can see place, that forces Clang/LLVM to use THIS function instead of implementing pop count as simple operations "by hand" ?